README.md

```diff
@@ -116,6 +116,20 @@ rather than a cluster:
 polaris --audit --audit-path ./deploy/
 ```
 
+##### Exit Codes for Audits
```

Comment:
The README is getting a little dense. What do you think about breaking this and the CLI flags into a separate file?